.github/workflows/deploy.yml

```
@@ -266,6 +364,21 @@ jobs:
           projects: ${{ vars.SENTRY_PROJECT_ARRAY }}
           disable_telemetry: true
           ignore_missing: true
+      - name: Add Monitoring Info to Summary
+        run: |
+          echo "## Monitoring" >> $GITHUB_STEP_SUMMARY
+          echo "- 📊 [Sentry Release](${{ vars.SENTRY_SERVER }}/organizations/${{ vars.SENTRY_ORG }}/releases/${{ vars.SENTRY_PREFIX }}@${{ needs.deploy-dev.outputs.version }}/)" >> $GITHUB_STEP_SUMMARY
+          echo "" >> $GITHUB_STEP_SUMMARY
+      - name: Add Related Changes to Summary
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.7)

property "version" is not defined in object type {deploy-dev: {outputs: {version: string}; result: string}}

(expression)